examples/wheel/wheel_test.py

```diff
@@ -394,11 +395,39 @@ def test_rule_sets_stamped_version_in_wheel_metadata(self):
             version = None
             with zf.open(metadata_file) as fp:
                 for line in fp:
-                    if line.startswith(b'Version:'):
+                    if line.startswith(b"Version:"):
                         version = line.decode().split()[-1]
             self.assertIsNotNone(version)
             self.assertNotIn("{BUILD_TIMESTAMP}", version)
 
+    def test_with_auto_add_requirements(self):
+        if sys.platform.startswith("win"):
+            self.skipTest("This test doesn't work on Windows")
+        filename = os.path.join(
+            os.environ["TEST_SRCDIR"],
+            "rules_python",
+            "examples",
+            "wheel",
+            "my_pytest_wheel-0.0.1-py3-none-any.whl",
+        )
+        with zipfile.ZipFile(filename) as zf:
+            metadata_file = None
+            for f in zf.namelist():
+                if os.path.basename(f) == "METADATA":
+                    metadata_file = f
+            self.assertIsNotNone(metadata_file)
+
+            requires = []
+            with zf.open(metadata_file) as fp:
+                for line in fp:
+                    if line.startswith(b"Requires-Dist:"):
+                        requires.append(line.decode().strip())
+            self.assertEqual(4, len(requires))
```

Comment:
nit: put the length check last

More preferable, assert against an expected list of all the values. This implicitly requires exactly equality, plus providers more context if there are errors.

Reply:
Addressed in ca608f6.